packages/friends/yml/en/john-spalding.yml

```diff
@@ -40,11 +40,16 @@ documents:
     slug: few-reasons-for-leaving
     filename: A_Few_Reasons_for_Leaving
     description:
-      A Few Reasons for Leaving the National Established Mode of Worship, Addressed to the
-      professors of religion in this day, by one who was long in the profession, but knew
-      not the power, till it pleased the Lord, by the ministry and writings of the people
-      called Quakers, to direct him to where alone the power is to be known, that is,
-      within.
+      Though raised in the Church of England, John Spalding's (1765-1795) pursuit of truth
```

Comment:
Above ^ there is still a `TODO` for the description of the other doc, the convincement and religious progress etc. I'm guessing that was an oversight. Probably some or most of the same text could be re-used for that doc as well.